app/styleguide/components/s-color-item.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div :class="b('grid-item')">
+    <div :class="b('color-wrapper')">
+      <div :class="b('color')" :style="backgroundColor"></div>
```

Comment:
I would recommend object syntax when using v-bind so it is easier to extend: https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax-1

Reply:
Fixed